code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -50,13 +50,24 @@
 /datum/chemical_reaction/plasmasolidification
 	name = "Solid Plasma"
 	id = "solidplasma"
-	required_reagents = list("iron" = 5, "frostoil" = 5, "plasma" = 20)
+	required_reagents = list("frostoil" = 5, "plasma" = 20)
```

Comment:
I believe that breathing plasma creates liquid plasma (or toxins) in the human. You may wish to test for that.